packages/genie/package.py

```diff
@@ -69,7 +70,13 @@ def url_for_version(self, version):
 
     depends_on("lhapdf" , when="+lhapdf")
 
-    depends_on("root+pythia6")
+    # Issues caused by default root cxxstd being 11
+    # Separate issue caused by libxmp from +x requirement
+    depends_on("root @6.24.08: +pythia6 ~x cxxstd=11", when="cxxstd=11")
```

Comment:
The pythia6 variant is removed in root 6.30.

Reply:
Ok, I can add a restriction on the versioning such that it does not apply that variant for 6.30 and beyond. Thanks!

Reply:
Using newer versions of root seems to not work with genie due to pythia6 issues. I am going to just restrict the version of root (for now) in the genie package.py. Looks like header files were removed from root at some point, similar to a larsim issue I submitted last week on redmine. Is this an issue that should be handed off to genie people? Does this fix seem reasonable?

[spack-build-out.txt](https://github.com/user-attachments/files/16714553/spack-build-out.txt)

Reply:
Seems OK to me.